### DISCUSSION TITLE
Scythe Anticheat v2.16.0

### DISCUSSION THREAD

**Discussion by MrDiamond64** [Scythe-Anticheat/Scythe-Anticheat#113] [Announcements]
Scythe Anticheat is now at version v2.156.0!

Supported Minecraft Versions: 1.20.30

# Changelog
- Add support for MC version 1.20.15
- XrayA now uses a Gametest implementation
- !ui now enchants the UI item again
- Made command handler support command registering
- Added "Configure Settings" menu to UI - modules can now be disabled/enabled in-game.
- Added "Server Settings" menu to UI
- Add Fly/B
- Added !deop - removes scythe op from players
- UI gamemode changer can now change players to spectator mode or the default gamemode
- Port !stats command to use Gametest APIs
- Autoclicker and Autoban no longer have a toggle command. Use the config instead.
- Add Spammer/E
- Fly/A now works again
- Disabled InvalidSprint/D due to false positives
- Added !globalmute - block all players except staff from sending chat messages
- Added helper gamemode commands, such as `!gms`, `!gma`, `!gmc` and `!gmsp`.
- Added welcome message module
- Added Aim/A
- Fix Tower/A false positives with shulker boxes, fences and walls.
- All modules now support "exclude_scythe_op" in its settings.
- Implement last good position mechanic - players flagged by movement checks will be teleported to the last known good position
- Fixed NoSlow/A false positives when eating while on ice
- `!invsee` now shows items in offhand and in armour slots
- Bug fixes
- Performance optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/Scythe-Anticheat/Scythe-AntiCheat/releases/tag/v2.16.0'>Scythe Anticheat v2.16.0</a>.</em>

**Comment by Lujo80x** [comment]
im on a realm and when i do /execute op i still cant do commands

**Reply by Lujo80x** [reply]
i mean function

**Reply by MrDiamond64** [reply]
Do you get any sort of error @Lujo80x?

**Reply by Lujo80x** [reply]
i hadnt updated minecraft and is there any feature to stop fly hacking?

**Comment by Justin-City-Games-And-More** [comment]
This released the EXACT day we were gonna switch from scythe anticheat but since this fixes all the issues  we had with the anticheat we dicided to stay.
